charts/prometheus-json-exporter/templates/deployment.yaml

```diff
@@ -57,6 +57,8 @@ spec:
           {{- with .Values.additionalVolumeMounts }}
             {{- toYaml . | nindent 12 }}
           {{- end  }}
+          env:
```

Comment:
By default, env is an empty list, in the values, which is fine. This will only fail if the user overwrites the environmentVariables with nothing on purpose.  The same is happening on line 52 for the resources, but if you really prefer, I'll change it 😄

Reply:
@monotek Please review again